root/static/scripts/common/components/FilterForm.js

```diff
@@ -104,6 +133,25 @@ const FilterForm = ({form}: Props): React.Element<'div'> => {
               </td>
             </tr>
 
+            {roleTypeField && roleTypeOptions ? (
+              <tr>
+                <td>
+                  {addColonText(l('Role'))}
```

Comment:
This label will show untranslated while hydrating until the pot files are regenerated.